components/NavBar.tsx

```diff
@@ -0,0 +1,70 @@
+"use client";
+import React from "react";
+import Image from "next/image";
+import AppBar from "@mui/material/AppBar";
+import Toolbar from "@mui/material/Toolbar";
+import Typography from "@mui/material/Typography";
+import { Stack } from "@mui/material";
+import Navigation from "@/constants/Navigation";
+import NavBarButtonVisita from "@/components/ui/NavBarButtonVisita";
+import { dosis } from "@/fonts/font";
```

Comment:
Here it was my mistake when I gave the example.
The right path is @/fonts only and not @/fonts/font (font would be the name of the font)